spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/credentials/transformer/DefaultCredentialEnvelopeTransformerRegistry.java

```diff
@@ -12,31 +12,28 @@
  *
  */
 
-package org.eclipse.edc.identityhub.processor.data;
-
-import org.eclipse.edc.identityhub.spi.processor.data.DataValidator;
-import org.eclipse.edc.identityhub.spi.processor.data.DataValidatorRegistry;
+package org.eclipse.edc.identityhub.spi.credentials.transformer;
 
 import java.util.HashMap;
 import java.util.Map;
 
-public class DataValidatorRegistryImpl implements DataValidatorRegistry {
+public class DefaultCredentialEnvelopeTransformerRegistry implements CredentialEnvelopeTransformerRegistry {
```

Comment:
Should be called `CredentialEnvelopeTransformerRegistryImpl` as I dont see any other implementation.